spec/linter-tslint-spec.js

```diff
@@ -47,4 +48,10 @@ describe('The TSLint provider for Linter', () => {
       }),
     );
   });
+
+  it('validates even when there is no tslint.json', () => {
+    waitsForPromise(() =>
+      atom.workspace.open(noConfigPath).then(editor => lint(editor)),
```

Comment:
Not really sure. In this test I don't care that the file was accurately validated, nor the specific results, I just want to make sure that validation completed